<ui version="4.0" >
 <class>TestWin</class>
 <widget class="QDialog" name="TestWin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Test window for OpenChrono</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>587</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>RPM</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="SliderRPM" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum" >
             <number>20000</number>
            </property>
            <property name="singleStep" >
             <number>10</number>
            </property>
            <property name="pageStep" >
             <number>100</number>
            </property>
            <property name="value" >
             <number>0</number>
            </property>
            <property name="sliderPosition" >
             <number>0</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lblRPM" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>0</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="StartStop" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Start / stop</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="butTest" >
          <property name="text" >
           <string>Test</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>T1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="SliderTemp1" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum" >
             <number>200</number>
            </property>
            <property name="singleStep" >
             <number>1</number>
            </property>
            <property name="pageStep" >
             <number>10</number>
            </property>
            <property name="value" >
             <number>49</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lblT1" >
          <property name="minimumSize" >
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>100</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>T2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="SliderTemp2" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum" >
             <number>200</number>
            </property>
            <property name="singleStep" >
             <number>1</number>
            </property>
            <property name="pageStep" >
             <number>10</number>
            </property>
            <property name="value" >
             <number>49</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lblT2" >
          <property name="minimumSize" >
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>100</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="2" >
           <widget class="QPushButton" name="butRight" >
            <property name="text" >
             <string>Right</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QPushButton" name="butLeft" >
            <property name="text" >
             <string>Left</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QPushButton" name="butDown" >
            <property name="text" >
             <string>Down</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QPushButton" name="butUp" >
            <property name="text" >
             <string>Up</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="butOk" >
            <property name="text" >
             <string>Ok</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>63</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="butCancel" >
            <property name="text" >
             <string>Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType" >
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QProgressBar" name="progressBar" >
        <property name="maximum" >
         <number>20000</number>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
        <property name="alignment" >
         <set>Qt::AlignHCenter</set>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="format" >
         <string>%v RPM</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons" >
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>600</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <widget class="QLabel" name="label_gx" >
      <property name="text" >
       <string>gx</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblgx" >
      <property name="text" >
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_gy" >
      <property name="text" >
       <string>gy</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblgy" >
      <property name="text" >
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QAccelerometer" name="accelerometer" >
   <property name="geometry" >
    <rect>
     <x>600</x>
     <y>0</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <zorder>accelerometer</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAccelerometer</class>
   <extends>QGraphicsView</extends>
   <header>qaccelerometer.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>SliderRPM</tabstop>
  <tabstop>StartStop</tabstop>
  <tabstop>SliderTemp1</tabstop>
  <tabstop>SliderTemp2</tabstop>
  <tabstop>butLeft</tabstop>
  <tabstop>butDown</tabstop>
  <tabstop>butRight</tabstop>
  <tabstop>butUp</tabstop>
  <tabstop>butOk</tabstop>
  <tabstop>butCancel</tabstop>
  <tabstop>butTest</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TestWin</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TestWin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderRPM</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lblRPM</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>35</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel" >
     <x>53</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderTemp2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lblT2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>262</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel" >
     <x>271</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderTemp1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lblT1</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>193</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel" >
     <x>173</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderRPM</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>33</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel" >
     <x>93</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
